dev/feature/3424dfb.unknown-connection-sqlexecutequery/iesi-dev.yml

```diff
@@ -0,0 +1,5 @@
+---
+type: Environment
+data:
+  name: "iesi-dev"
+  description: "iesi-dev environment"
```

Comment:
Are these files added to the repo as test or example scripts for documentation?

Reply:
These are the files to test the implementation

Reply:
Okay, but can we maybe group these test files under the /test folder? Creating a /dev folder for each feature will cause the repository to explode